umm-lib/src/cmr/umm/echo10/granule.clj

```diff
@@ -57,13 +57,12 @@
                   size-unit
                   size-in-bytes
                   checksum]} data-granule
-          day-night (if day-night day-night "UNSPECIFIED")
-          size-mb (get-size-in-mb size size-unit)]
+          day-night (if day-night day-night "UNSPECIFIED")]
       (x/element :DataGranule {}
                  (when size-in-bytes
                    (x/element :DataGranuleSizeInBytes {} size-in-bytes))
                  (when size
-                   (x/element :SizeMBDataGranule {} size-mb))
+                   (x/element :SizeMBDataGranule {} (get-size-in-mb size size-unit)))
```

Comment:
I don't see any real code change in this file. Is this ticket OBE?

Reply:
No, the code change is moving the (get-size-in-mb size size-unit) function from the let statement to be wrapped in a when clause after line 64.  This prevents the function getting called if the size is nil, whereas before it was being called and could cause a null pointer exception.

Reply:
Got it. I missed the `when` clause.